rootfs/templates/wrapper

```diff
@@ -84,13 +84,49 @@ function use() {
     # Implement DNS fix related to https://github.com/docker/docker/issues/24344
     DOCKER_ARGS+=("--dns=${DOCKER_DNS}")
   fi
-
+     
   if [ -n "${HOME}" ]; then
     if [ "${HOME}" == "/localhost" ]; then
       echo "WARNING: not mounting ${HOME} because it conflicts with geodesic"
     else
-      echo "# Mounting ${HOME} into container"
-      DOCKER_ARGS+=(--volume=${HOME}:/localhost)
+      if  [[ $(cat /proc/version) =~ Microsoft ]] ; 
+      then
+        if [ -z "${LOCAL_HOME}" ]; 
```

Comment:
The logic should be like this:

```
if [ -z "${LOCAL_HOME}" ]; then
... now try to figure it out...
if WSL, and we can't find HOME, error. 
otherwise, LOCAL_HOME=${HOME}
fi
...

DOCKER_ARGS+=(--volume=${LOCAL_HOME}:/localhost)

```